### DISCUSSION TITLE
Bambu Lab PETG-HF filament

### DISCUSSION THREAD

**Discussion by DarrenMut** [OrcaSlicer/OrcaSlicer#6784] [Q&A]
Hi, I am new to Orca Slicer and have just installed that latest version on windows

I have been able to attach to my Bambu Lab X1 Carbon and things appear to be okay, I have been keen to learn and use Orca Slicer for the calibration tools

My question I am hoping someone could help me with is with regards to Filaments, I have ensured all filaments are loaded when I went through the setup wizard. But I am not seeing or able to add the new Bambu Lab PETG-HF filament which I use all the time. So its not being detected by AMS or is a option if I add filaments

Could someone please advise if this is available or likely to be added as a default in the near future?

I have done some searches on this problem but not found anything that helps with this problem or explains how i can add without creating a hole new filament profile. I am guessing i am not only person using this filament

Any advise or links to previous answers would be greatly appreciated, sorry if I am missing something simple. I really want to step away from Bambu Studio (which has the PETG-HF) to the Orca Slicer as I believe its a better Slicer

Thanks in advance of any advise or support
Darren

**Comment by LBowers23** [comment]
Have you figured this out yet?   I haven't bought any PETG-HF yet as I was waiting until the profiles were available in Orca (I'm running 2.1.1).  I see in the Bambu Studio (1.9.3.50) I have installed it is showing there.

Larry

**Comment by jc3d-melb** [comment]
Bambu PETG HF is included in the new release candidate

**Comment by DarrenMut** [comment]
Great thanks for the feedback, time to download i guess

**Comment by LBowers23** [comment]
Thank you for the update.  I'll wait for the stable release.  I'm in no rush at present, but just wondering.  Appreciate all the effort the developers have made to this product.

Larry
